client/src/Components/Tasks/TaskAdd.js

```diff
@@ -0,0 +1,176 @@
+      endDate: null,
+      assignedTo: null,
+
+    }
+  }
+
+  //methods here
+  componentDidMount() {
+    this.props.fetchEmployees();
+    this.props.fetchProperties();
+    this.props.fetchReservations();
+  };
+
+  handleChange = (prop, val) => {
+    this.setState({ [prop]: val });
+  };
+
+  handleDateChange = ({ startDate, endDate }) => {
+    this.setState({ startDate: startDate, endDate: endDate });
+  };
+
+  handleSubmit = () => {
+    this.props
+      .createTask(this.state)
+      .then(data => {
+        if (data._id) {
+          this.props.history.push("/dashboard/tasks");
+        }        
+      })
+      .catch(err => {});
+  };
+
+  render() { 
+
+    // const { description } = this.state;
+
+    return ( 
+      <FlexColumn>
+        <FlexRow>
+          <Header as="h1">Add Tasks</Header>
+        </FlexRow>
+
+        <br />
+
+        <FlexColumn>
+
+          <FlexRow>
+            <Input
+              placeholder="Add Task"
+              onChange={e =>
+                this.handleChange("description", e.target.value)
+              }
+            />
+          </FlexRow>
+
+          {/* <br />
+
+          <FlexRow width="100%" justifyCenter>
+            <Button color='teal'>Add Task</Button>
+          </FlexRow> */}
+
+        </FlexColumn>
+
+        <br />
+
+        {/* <FlexRow width="full" justifyCenter>
+          <Segment>       
+            <FlexColumn width="full" justifyCenter> */}
+              {/* map over tasks as they are created */}
+              {/* <p>Task Item 1</p>
+              <p>Task Item 2</p>
+              <p>Task Item 3</p>
+            </FlexColumn>
+          </Segment>
+        </FlexRow> */}
+
+        <FlexRow width='full'>
+          <DateRangePickerWrapper onChange={this.handleDateChange} />
+        </FlexRow>
+
+        <br />
+        <br />
+        {console.log(this.props)}
+
+        <FlexRow>
+          <Dropdown 
+            placeholder="Property"
+            style={{ marginRight: "10px" }}
+            selection
+            onChange={(e, val) => this.handleChange("property", val.value)}
+            options={
+              this.props.loading
+                ? [{ text: "Loading...", value: "loading" }]
+                : this.props.tasks.properties &&
+                  this.props.tasks.properties.map(p => ({
+                    key: p._id,
+                    text: p.name,
+                    value: p._id
+                  }))
+            }
+          />
+        </FlexRow>
+
+        <br />
+
+        <FlexRow>
+          <Dropdown
+            placeholder="Reservation"
+            style={{ marginRight: "10px" }}
+            selection
+            onChange={(e, val) => this.handleChange("reservation", val.value)}
+            options={
+              this.props.loading
+                ? [{ text: "Loading...", value: "loading" }]
+                : this.props.tasks.employees &&
+                  this.props.tasks.employees.map(r => ({
```

Comment:
I don't know what I did wrong in my Redux, but to get the reservations, I had to do `tasks.employees.map`. That is how the data was stored in my Redux DevTools. If you can figure it out, let me know!!!!